packages/now-cli/test/unit.js

```diff
@@ -56,7 +58,22 @@ const getStaticFiles = async (dir, isBuilds = false) => {
     strict: false,
   });
 
-  return getStaticFiles_(dir, nowConfig, { hasNowJson, output, isBuilds });
+  const files = await getStaticFiles_(dir, nowConfig, {
+    hasNowJson,
+    output,
+    isBuilds,
+  });
+  return normalizeWindowsPaths(files);
+};
+
+const normalizeWindowsPaths = files => {
+  if (process.platform === 'win32') {
+    const prefix = 'D:/a/now/now/packages/now-cli/test/fixtures/unit/';
```

Comment:
What is this path?

Reply:
For some reason, Windows is returning absolute paths instead of relative paths.

This was my hack to turn it into a relative path.